community/knative/charts/serving/charts/monitoring/charts/elasticsearch/Chart.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+name: elasticsearch
+version: 0.1.0
+description: Elasticsearch logging for Knative monitoring
+appVersion: "0.2.3"
+icon: https://raw.githubusercontent.com/knative/docs/master/images/logo/rgb/knative-logo-rgb.jpg
+kubeVersion: ">=1.10"
+tillerVersion: ">=2.10.0"
```

```suggestion
tillerVersion: ">=2.9.1"
```
Needs to support at least 2.9.1 for ICP 3.1.x support.

---

same as [#184 (comment)](https://github.com/IBM/charts/pull/184#discussion_r255170249)